api/draupnir-openapi.yaml

```diff
@@ -0,0 +1,627 @@
+openapi: 3.1.0
+info:
+  title: Draupnir API
+  description: The API for the Draupnir bot
+  version: 0.1.0
+  license:
+    name: AFL-3.0
+    url: https://github.com/the-draupnir-project/Draupnir/blob/main/LICENSES/AFL-3.0.txt
+paths:
```

Comment:
Will it be easy to split this up into different files later?

Reply:
All openapi rules apply so yeah you can use cross file ref stuff https://swagger.io/docs/specification/v3_0/using-ref/

But also it uses the tag thingy to generate the sidebar folders.

Reply:
okie